src/events/onStateChangedEmitter.ts

```diff
@@ -0,0 +1,7 @@
+import { EventEmitter } from "vscode";
+import { State } from "../binary/state";
+
+const onStateChangedEmitter = new EventEmitter<State>();
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
y like dis? y note `export default`?

Reply:
`export default` is hell
`https://www.lloydatkinson.net/posts/2022/default-exports-in-javascript-modules-are-terrible/`